nav2_rviz_plugins/src/nav2_panel.cpp

```diff
@@ -17,9 +17,12 @@
 #include <QtConcurrent/QtConcurrent>
 #include <QVBoxLayout>
 #include <QHBoxLayout>
+#include <QTableWidgetItem>
 #include <QTextEdit>
 #include <QCheckBox>
 
+
+#include <array>
```

Comment:
this too

Reply:
Does NAV2 intend to use transitive includes? I am using array in the header now.

Reply:
Interesting.. I think array is already part of the standard library and there in no necessity to add it here, since I guess the std library is already added. It should have been the case with vector too. We need memory because it is not part of the standard library. 

Maybe @SteveMacenski can shed some light here..

Reply:
@SteveMacenski Please advise on the use of transitive includes in NAV2? 

Google style guide says not to reply on them, but @padhupradheep says my includes are not necessary. 
https://google.github.io/styleguide/cppguide.html#Include_What_You_Use

Reply:
You need to have it if you use it.